manifests/0000_21_cluster-openshift-controller-manager-operator_01_config.crd.yaml

```diff
@@ -13,4 +13,18 @@ spec:
     categories:
     - coreoperators
   subresources:
-    status: {}
\ No newline at end of file
+    status: {}
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
> This operator is responsible for creating the `config.Build` CRD.

This is not the correct location.  We want as little as possible managed by the CVO because it significantly constrains future choice.  If you need to have the operator create a CRD, let the operator make a CRD.

Reply:
/cc @openshift/sig-master in case it comes up in other operators.  We don't want to do this.

Reply:
@deads2k please elaborate on how this constrains future choice so we can understand the argument you are making.  From my perspective this is defining a resource name/group, nothing more, and a future operator could delete it entirely if that were deemed necessary, so writing code to do it instead of dropping a yaml file in a directory is unnecessary work against an unclear future risk.


/cc @derekwaynecarr @jwforres

Reply:
https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/pkg/operator/sync_openshiftcontrollermanager_v311_00.go#L32-L43 is a good spot to do it.

The CVO removes items that are no longer in the payload.  Doing it our sync loop allows ownership to change in the future.  Putting it in the CVO payload pins it for all time (or until someone builds a special migrator)

Reply:
@deads2k and I spoke in his cube... the way forward for this is:

1) add the CRD to openshift/api, ala: https://github.com/openshift/api/pull/118
2) vendor the .yaml in from openshift/api to this repo
3) add logic in the operator that installs the yaml, ala https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/pkg/operator/sync_openshiftcontrollermanager_v311_00.go#L32-L44  (not sure this is precisely where this particular CRD would go since it is not v311 related though, probably need more specific insight from @deads2k about where he wants to see it).